method/resources/Transaction.py

```diff
@@ -0,0 +1,30 @@
+from typing import TypedDict, List
+
+from method.resource import Resource
+from method.configuration import Configuration
+
+
+class Transaction(TypedDict):
+    id: str
+    acc_id: str
+    mcc: str
+    description: str
+    presentable_description: str
+    amount: str
+    currency: str
+    billing_amount: str
+    billing_currency: str
+    status: str
+    created_at: str
+    updated_at: str
+
+
+class TransactionResource(Resource):
+    def __init__(self, config: Configuration):
+        super(TransactionResource, self).__init__(config.add_path('resource'))
```

Comment:
is this correct? 
`config.add_path('resource')`

Reply:
@ossiggy – should be config.add_path('transactions')